apis/v1alpha2/httproute_types.go

```diff
@@ -705,17 +705,25 @@ type HTTPPathModifier struct {
 	// Type defines the type of path modifier.
 	//
```

Comment:
I think we need a callout here that we expect new values to be added and that implementations should ensure they handle values not in this list without crashing etc.

Reply:
Updated, thanks!